.github/workflows/manpage.yml

```diff
@@ -0,0 +1,29 @@
+name: Building man page and man.test
+on:
+  push:
+    paths:
+      - 'docs/**'
+      - '.github/workflows/manpage.yml'
+
+jobs:
+  website:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+          cache: pipenv
+      - name: Install pipenv
+        run: pip install pipenv
+      - name: Install dependencies
+        run: pipenv sync
+        working-directory: docs
+      - name: Build man page and man.test
+        run: |
+          rm jq.1.prebuilt tests/man.test
+          make jq.1.prebuilt tests/man.test
+      - name: Make sure tests are up to date
+        run: git diff tests/man.test
```

Comment:
Drop `--quiet` and use `--exit-code` instead, to make it understandable what happened on diff failure.

Reply:
Oh, I already dropped `--quiet`, but I didn't know I needed to use `--exit-code`, thank you.